codecov.yml

```diff
@@ -0,0 +1,23 @@
+ignore:
+- "Tests"
+
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
I assume this force the code coverage to be between 70-100? Otherwise the PR won't be mergeable or what is the intention?

Reply:
its just the default

Reply:
> So this is a followup to #49 right?
> It basically adds the configuration file for codecov?

yes. Codecov didn't ignore the tests by itself, so I had to add this file

Reply:
Does the CI run enforce a certain code coverage? Or does the CI job just fail when the code coverage decreases like in this case?

Reply:
it just fails. Codecov checks are not "required"